.circleci/config.yml

```diff
@@ -0,0 +1,22 @@
+# Python CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-python/ for more details
+#
+version: 2
+jobs:
+  build:
+    docker:
+      - image: mbedos/mbed-os-env:stable
+    working_directory: ~
+    steps:
+      - checkout:
+          path: mbed-os-example-ble
+      - run: |
+          git clone https://github.com/ARMmbed/mbed-os.git
```

Comment:
We expect master to always be compatible with the head of the repo. 
But it doesn't mean the release branch will be compatible with the head of the repo in case of breaking change.